tensorflow/python/ops/math_ops.py

```diff
@@ -3212,6 +3222,15 @@ def sigmoid(x, name=None):
 
   Returns:
     A Tensor with the same type as `x`.
+    
+  Usage Example:
+
+  >>> y = tf.math.sigmoid(1.0)
+  >>> print(y)
+  tf.Tensor(0.73..., shape=(), dtype=float32)
+  >>> y = tf.math.sigmoid(-1.0)
+  >>> print(y)
+  tf.Tensor(0.26..., shape=(), dtype=float32)
```

Comment:
I'm wondering if the doctest automatically stops checking after the ..., in which case the rest of the line isn't needed. Not sure though.

Reply:
Nice to see another GCI participant here! I've run this code through the checks before, and it seemed to have worked, so I don't believe that's an issue. I'll look into changing your other requests when I get the chance!

Reply:
Please consult https://www.tensorflow.org/community/contribute/docs_ref regarding the `...`

Reply:
I did, and their example has this:
``` <tf.Tensor: shape=(1,), dtype=float32, numpy=..., dtype=float32)> ```
They still kept the rest of the line after the ellipsis.

Reply:
Yes, but the float is not trimmed.

Reply:
One of the previous reviewers left a comment about this:

> Use smaller numbers like 1 and -1 that let you use fewer digits of precision (so replace 0.99999546 with 0.9... etc)

And as such, I believe it is still okay if there is a little bit of the number remaining (this also passed all the checks). In addition, I'd prefer it if at least the first digit of the float remained, as my explanation displays the relationship between the output of the sigmoid function and the probabilities.

Let me know if you have any more suggestions.

Reply:
How about

```
  >>> y = tf.math.sigmoid(0.0)
  >>> print(y)
  tf.Tensor(0.5, shape=(), dtype=float32)
```

The float is fully written out (as we want it to be), easy to compute

Reply:
Good idea! Is it okay if I keep my original numbers as well (since I wanted to show how it approaches 1 and approaches 0)?

Reply:
let's try very large numbers then (`-128` and `128` maybe) where it would be `0` and `1`

Reply:
That's a good idea, but I feel like the way it stands right now is fine too, right?